php/src/Google/Protobuf/Internal/Message.php

```diff
@@ -1260,7 +1273,13 @@ private function fieldDataOnlyJsonByteSize($field, $value)
                 $size += 2;  // size for \"\"
                 break;
             case GPBType::MESSAGE:
-                $size += $value->jsonByteSize();
+                if ($field->isTimestamp()) {
```

Comment:
Can we merge this logic to jsonByteSize?

Reply:
That's done, I've made the same change with `serializeToJsonStream`